data/gaming.html

```diff
@@ -117,9 +117,9 @@ <h2><a onclick="cmd('link?https://deltarune.com/')"><span
                                     </h2>
                                     <div class="card card-block">
                                         <div class="card-image text-md-center">
-                                            <a href="img/screenshots/DELTARUNE.jpg" data-toggle="lightbox"><img
+                                            <img
```

Comment:
and so on for all the remaining img changes

Reply:
formatted